src/slice.cr

```diff
@@ -239,6 +239,68 @@ struct Slice(T)
     pointer(count).copy_to(target, count)
   end
 
+  # Copies the contents of this slice into *target*.
+  #
+  # Truncates if this slice doesn't fit into the destination slice.
```

Comment:
And yes, i'm using it [here](https://github.com/crystal-lang/crystal/pull/2973/commits/95914ebd46a2a809807ec153d357a25bfd0fcd2f#diff-2ec648b622c7b4b10834b530a78ce526R77).